.gitmodules

```diff
@@ -1,4 +1,4 @@
-[submodule "eth2near/ethashproof"]
-	path = eth2near/ethashproof
+[submodule "ethashproof"]
+	path = ethashproof
```

Comment:
Should we move it back into `eth2near`? :smile:

Reply:
++